README.md

```diff
@@ -76,7 +77,8 @@ if err != nil {
     panic(err)
 }
 ``` 
-The encoded bytes of the struct from above example will look like:
+
+The encoded bytes of the struct in the example above:
```

Comment:
The encoded bytes of the struct MyUser is as following:

Reply:
should be `as follow`